tests/bin/bump-sentry

```diff
@@ -1,65 +0,0 @@
-#!/usr/bin/env python
```

Comment:
This file ended up not being used by tests. I don't know if it ever did.

Either way, it's out of date.